### PR TITLE
Add description field to kernelInfo

### PR DESCRIPTION
**Pull request by colombod**
This Pr adds support for description text on `KernelInfo`.
The description property is forwarded to `ProxyKernels` too.

Removed unused code with mermaid extension.

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs [hunk 1]
```diff
@@ -61,6 +61,10 @@ public CSharpKernel(string name) : base(name)
         KernelInfo.LanguageName = "C#";
         KernelInfo.LanguageVersion = "12.0";
         KernelInfo.DisplayName = $"{KernelInfo.LocalName} - C# Script";
+        KernelInfo.Description = """
+                                 This Kernel can compile and execute C# code and display the results.
+                                 The language is C# Scripting, a dialect of C# that is used for interactive programming.
+                                 """;
```
Comment (shyamnamboodiripad) [inline src/Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs:67]: Is this displayed anywhere in the UI? Will we need to localize this description for users who are on other languages?

**Comment by shyamnamboodiripad** [comment]
Should we add a description for the composite kernel created by the dotnet-interactive tool?

**Comment by colombod** [comment]
No it is not displayed at all

Dr Diego Colombo
(he/him)
Principal software engineer
.NET Interactive <https://github.com/dotnet/interactive>
________________________________

**Review by shyamnamboodiripad — APPROVED** [review]

**Comment by shyamnamboodiripad** [comment]
> ... I added already the description to the composite kernel in the extension and on js and html kernels too

Ah I was referring to the composite kernel that is created in [this method](https://github.com/dotnet/interactive/blob/main/src/dotnet-interactive/CommandLine/CommandLineParser.cs#L454C3-L454C49). Not super important though - we can always add it later.

**Comment by colombod** [comment]
That composite kernel (like the one in the dotnet tool) doesn't do anything special besides routing, there is no need for custom description.

**Pull request merged**